docs/topics/settings.rst

```diff
@@ -218,6 +218,25 @@ Default: ``None``
 
 The name of the region associated with the AWS client.
 
+.. setting:: ASYNCIO_LOOP
+
+ASYNCIO_LOOP
+------------
+
+Default: ``None``
+
+Import path of a given asyncio loop.
```

Comment:
I think there is an inconsistency here. Personally, I read this setting's description as if it were the import path of the event loop _class_, but this actually expects the _module_. It is documented as such in the above "Using custom asyncio loops" section, but to be consistent with the `TWISTED_REACTOR` setting I think it should be the path of the class instead.

Also, would `ASYNCIO_EVENT_LOOP` be a better name? Not a strong suggestion, just thinking out loud.

Reply:
I am changing the setting to expect the class path for better consistency. Also changing the setting name to ASYNCIO_EVENT_LOOP